stdlib/file/src/test/java/org/ballerinalang/stdlib/file/FileTest.java

```diff
@@ -121,7 +121,7 @@ public void testFileRenameWithInvalidPath() throws IOException {
             BValue[] returns = BRunUtil.invoke(compileResult, "testRename", args);
             assertTrue(returns[0] instanceof BError);
             BError error = (BError) returns[0];
-            assertEquals(error.getReason(), FileConstants.INVALID_OPERATION_ERROR);
+            assertEquals(error.getMessage(), FileConstants.INVALID_OPERATION_ERROR);
```

Comment:
Ideally FileConstants.INVALID_OPERATION_ERROR should not be retrieved as the error message. The message should be `File already exists in the new path `